tests/performance/log-results.js

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+const https = require( 'https' );
+const [ token, branch, hash, timestamp ] = process.argv.slice( 2 );
+
+const resultsFiles = [
+	{
+		file: 'home-block-theme.test.results.json',
+	},
+	{
+		file: 'home-classic-theme.test.results.json',
+	},
+];
+
+const performanceResults = resultsFiles.map( ( { file } ) =>
+	JSON.parse(
+		fs.readFileSync( path.join( __dirname, '/specs/' + file ), 'utf8' )
+	)
+);
+
+function median( array ) {
+	const mid = Math.floor( array.length / 2 ),
+		numbers = [ ...array ].sort( ( a, b ) => a - b );
+	return array.length % 2 !== 0
+		? numbers[ mid ]
+		: ( numbers[ mid - 1 ] + numbers[ mid ] ) / 2;
+}
+
+const rawResults = [];
+for (var keys in performanceResults) {
+    const rawKeys = [];
+    for (var key in performanceResults[keys]) {
+        rawKeys[key] = median( performanceResults[keys][key] );
+    }
+    rawResults.push( rawKeys );
+}
+
+const data = new TextEncoder().encode(
+	JSON.stringify( {
+		branch,
+		hash,
+        baseHash: '',
+		timestamp: parseInt( timestamp, 10 ),
+		metrics: resultsFiles,
+        baseMetrics: '',
```

Comment:
Looks like some spacing issues here:

```suggestion
		branch,
		hash,
		baseHash: '',
		timestamp: parseInt( timestamp, 10 ),
		metrics: resultsFiles,
		baseMetrics: '',
```

Reply:
Thanks @joemcgill Addressed in https://github.com/10up/wordpress-develop/pull/30/commits/42b676121e931ef46b999ac7a8479892065aa496